src/components/grid-aware/ThreeParagraphBlock/ThreeParagraphBlock.module.css

```diff
@@ -62,9 +62,12 @@
   column-gap: var(--grid-column-gap);
   display: grid;
   grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+  padding: 100px 0;
+}
+
+/* Grid parent for optional ctaButtons block */
+.gridParent.v2 {
```

Comment:
Is there a way for us to get rid of this variant altogether? I think we could write the CSS in a way such that just the presence or absence of the `<GridBottomArea>` will be sufficient for laying it out properly.

I think we can do this by getting rid of the `row-gap` and just having the `gridAreaBottom` set a top margin. That way, if it doesn't exist, then the extra spacing isn't there.